eZ/Publish/Core/Persistence/Legacy/User/Gateway/DoctrineDatabase.php

```diff
@@ -55,9 +55,6 @@ public function createUser(User $user)
                 $this->handler->quoteColumn('login'),
                 $query->bindValue($user->login)
```

Comment:
Shouldn't we here - when creating a new user - also make `$user->login` lowercase (as it was done for `login_normalized` before)?

If I insert a new user e.g. 'Andrew' to a postgresql database, creating a case in-sensitive index will not make login match 'andrew'. Unless I'm missing something or it is handled elsewhere...